src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
```

Comment:
let's create a constant NUMBER_OF_BALLS for the 3 number (now 3 is a magic number)
read more about constants here - https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names